internal/controller/scheduler/scheduler.go

```diff
@@ -19,6 +21,8 @@ import (
 
 	"github.com/buildkite/agent/v3/clicommand"
 
+	"github.com/distribution/reference"
```

Comment:
Handy library, have starred that one ⭐

Reply:
I tried finding one among the k8s libraries, and discovered they were using this under the hood.